### PR TITLE
chore: add simple diff tool in gitconfig file

### LINKED ISSUE #41: Add proper .gitconfig
We should add a proper `.gitconfig` to this repository to avoid this effect:

https://github.com/nearform/optic-release-automation-action/pull/39/files#diff-3d2b59189eeedc2d428ddd632e97658fe310f587f7cb63b01f9b98ffc11c0197

The line ending has changed

### PR DESCRIPTION
**Pull request by TomasTM**
closes #41 

Implement basic .gitconfig file to better manage line endings

### CODE REVIEW

**Comment by TomasTM** [comment]
@Eomm we can get pretty fancy with the config but this should be the bare minimum. let me know if you want to go crazy with [delta] or [color]

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[diff]
```
Comment (Eomm) [inline .gitconfig:1]:
```suggestion
[core]
	autocrlf = input
[diff]
```

**Review by Eomm — COMMENTED** [review]
for that issue, I was thinking more about the line endings.

the build executed by this action has committed an unchanged file:

https://github.com/nearform/optic-release-automation-action/pull/39/files#diff-3d2b59189eeedc2d428ddd632e97658fe310f587f7cb63b01f9b98ffc11c0197

modifying only the line endings, so I suppose it is an `autocrlf` issue.

The other `diff*` settings are more personal in my opinion

File: .gitconfig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[core]
+	autocrlf = input
+[diff]
+	tool = vimdiff
+[difftool]
+	prompt = false
```
Comment (simoneb) [inline .gitconfig:6]: are these needed? I have concerns they're portable cross platform

Comment (TomasTM) [reply .gitconfig:6]: Humm yeah that is a good point! I'll remove the tool and leave the prompt so that the confirmation doesn't appear

**Comment by simoneb** [comment]
Can we please make sure to **always** describe what the PR does and link the issue please?

**Review by Eomm — APPROVED** [review]

**Review by simoneb — CHANGES_REQUESTED** [review]
blocking until we have a PR description and a linked issue

**Review by simoneb — APPROVED** [review]

**Pull request merged**